platform/kobo/disable-wifi.sh

```diff
@@ -9,9 +9,9 @@ ifconfig eth0 down
 
 # Some sleep in between may avoid system getting hung
 # (we test if a module is actually loaded to avoid unneeded sleeps)
-if lsmod | grep -q $WIFI_MODULE ; then
+if lsmod | grep -q "$WIFI_MODULE" ; then
```

Comment:
Actually I think I prefer without curly braces because they're visual noise, although I don't feel very strongly about it. :-P